orga/tests/unit/routes/authenticated/team_test.js

```diff
@@ -1,11 +0,0 @@
-import { module, test } from 'qunit';
-import { setupTest } from 'ember-qunit';
-
-module('Unit | Route | authenticated/team', function (hooks) {
-  setupTest(hooks);
-
-  test('it exists', function (assert) {
-    const route = this.owner.lookup('route:authenticated/team');
-    assert.ok(route);
```

Comment:
le it exist, ne sert à rien. autant le supprimer. et rajouter plus tard des tests si besoin sur cette route

Reply:
Tu t'auto review ? :D